src/constants.js

```diff
@@ -11,10 +11,13 @@
 // the License.
 
 // SAMPLED_MASK is the bit mask indicating that a span has been sampled.
-export const SAMPLED_MASK = 0x1;
+export const SAMPLED_MASK = 0b1;
```

Comment:
We decided to use bit 4 for firehose, and reserve bit 3 for deferred sampling if we build it in the future.

Reply:
SGTM